src/components/vm/disks/vmDisksCard.jsx

```diff
@@ -267,12 +267,24 @@ export class VmDisksCard extends React.Component {
             };
 
             const deleteDialogProps = {
+                title: disk.device === "cdrom" ? _("Remove disc from VM?") : _("Remove disk from VM?"),
                 actionName: _("Remove"),
-                title: cockpit.format(_("Remove disk $0"), disk.target),
-                errorMessage: cockpit.format(_("Disk $0 could not be removed"), disk.target),
+                errorMessage: disk.device === "cdrom"
+                    ? cockpit.format(_("Disc $0 could not be removed"), disk.target)
+                    : cockpit.format(_("Disk $0 could not be removed"), disk.target),
```

Comment:
[My dictionary](https://dict.leo.org/englisch-deutsch/disc) says that disc vs. disk are mostly just American vs. British English. Duplicating these strings seems a bit dubious and unnecessary, especially as they are all in the same "Disks" table on the VM details anyway.

Reply:
@garrett mind giving input?

Reply:
Yeah, but it seems there are specific cases where it's used independed of variation of english. And that is optical discs:
1. `Optical disc image` or in short: `ISO`
It sems that even wikipedia calls it "disk" when it's a disk device in general, but "disc" when talking about ISO/CD: An optical **disc** image (or ISO image) is a **disk** image. [wiki](https://en.wikipedia.org/wiki/Optical_disc_image)
3. CD: `Compact disc read-only memory`
4. DVD: `Digital Versatile Disc` or `Digital Video Disc`

And when we are talking about ISO images in VMs, we are talking about emulations of Optical Disc image or CD.

Reply:
Right, I'm aware of that technical distinction -- but it doesn't make much difference here in the dialog IMHO. The VM details page has them all under the "Disks" title.

Reply:
Alright, reverted